shared/src/business/useCases/document/getDocumentContentsForDocketEntryInteractor.js

```diff
@@ -0,0 +1,32 @@
+/**
+ * getDocumentContentsForDocketEntryInteractor
+ *
+ * @param {object} providers the providers object
+ * @param {object} providers.applicationContext the application context
+ * @param {object} providers.documentContentsId document contents id
+ * @returns {string} url for the generated document on the storage client
+ */
+exports.getDocumentContentsForDocketEntryInteractor = async ({
+  applicationContext,
+  documentContentsId,
+}) => {
+  try {
```

Comment:
Should there be some permissions checks in here to verify that the user is allowed to see this content?

Reply:
Good idea, there probably should be!

Reply:
Should there be more permission checks here, the same as in `getCaseInteractor`? The way this is written now, anyone with petitioner credentials and a docketEntryId could get the full text of a document. Maybe that's not a concern, just thought I should throw it out there.